generator/src/routes/+page.svelte

```diff
@@ -315,7 +315,9 @@
             read using a compatible reader application, enabling access to relevant product details
             in real time.
           </p>
-          <a class="documentation" href={settings.DOCUMENTATION_URL}>See documentation →</a>
+          <a class="documentation" href={settings.DOCUMENTATION_URL} target="_blank"
```

Comment:
Task in Trello also asks for `ref="noreferrer"` (goes for all `a` that been edited). And formatting like on this particular `a` should be fixed.

Reply:
looks like it should be `rel`)